engine/src/main/java/org/destinationsol/game/SaveManager.java

```diff
@@ -191,4 +190,70 @@ public static ShipConfig readShip(HullConfigManager hullConfigs, ItemManager ite
 
         return new ShipConfig(hull, itemsStr, money, 1, null, itemManager, spawnPos);
     }
+
+    /**
+     * Saves the world to a file. Currently stores the seed used to generate the world and the number of systems
+     * @param numberOfSystems
+     */
+    public static void saveWorld(int numberOfSystems) {
+        Long seed = SolRandom.getSeed();
+        String fileName = SaveManager.getResourcePath(WORLD_SAVE_FILE_NAME);
+
+        JsonObject world = new JsonObject();
+        world.addProperty("seed", seed);
+        world.addProperty("systems", numberOfSystems);
+
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        String stringToWrite = gson.toJson(world);
+
+        PrintWriter writer = null;
+        try {
+            writer = new PrintWriter(fileName, "UTF-8");
+            writer.write(stringToWrite);
+            logger.debug("Successfully saved the world file");
+        } catch (FileNotFoundException | UnsupportedEncodingException e) {
+            logger.error("Could not save world file", e);
+            return;
+        } finally {
+            if(writer != null) {
+                writer.close();
+            }
+        }
+    }
+
+    /**
+     * Load the last saved world from file, or returns null if there is no file
+     */
+    public static WorldConfig loadWorld() {
+        if(SaveManager.resourceExists(WORLD_SAVE_FILE_NAME)) {
```

Comment:
Trivial whitespace violation (space between `if` and opening `(` - might want to make sure you've got Checkstyle set up 👍